### PR TITLE
Convert all projects to SDK

### PR DESCRIPTION
**Pull request by mansellan**
DRAFT!

### CODE REVIEW

**Comment by Greedquest** [comment]
Will this allow using `dotnet build` in CI instead of `msbuild`? How about other dotnet cli methods https://learn.microsoft.com/en-us/dotnet/core/tools/ ? @retailcoder @bclothier 
image

**Comment by mansellan** [comment]
> Will this allow using `dotnet build` in CI instead of `msbuild`? How about other dotnet cli methods https://learn.microsoft.com/en-us/dotnet/core/tools/ ? @retailcoder @bclothier

Yep, it very much should do. I just need to finish it off and get it building...

File: Client/Rubberduck.Main/Rubberduck.Main.csproj
Comment (Greedquest) [inline Client/Rubberduck.Main/Rubberduck.Main.csproj:1]: @retailcoder you [said](https://discord.com/channels/1074165523102826508/1074165722541981696/1095469521592143872) "RD.Main would have to remain on MSBuild until it's made into a SDK project." Looks like this PR makes that change if I understand correctly? Looks like _everything_ is SDK?

File: Client/Rubberduck.Main/Rubberduck.Main.csproj [hunk 1]
```diff
@@ -1,226 +1,27 @@
...
-    <None Include="NLog.dll.nlog">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="Ducky.ico" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (Greedquest) [inline Client/Rubberduck.Main/Rubberduck.Main.csproj:1]: @retailcoder you [said](https://discord.com/channels/1074165523102826508/1074165722541981696/1095469521592143872) "RD.Main would have to remain on MSBuild until it's made into a SDK project." Looks like this PR makes that change if I understand correctly? Looks like _everything_ is SDK?

Comment (retailcoder) [reply Client/Rubberduck.Main/Rubberduck.Main.csproj:1]: Yep, that's exactly it!

**Pull request closed without merging**